config/crd/skiperator.kartverket.no_applications.yaml

```
@@ -553,6 +553,83 @@ spec:
                     maximum: 3600
                     minimum: 1
                     type: integer
+                  authentication:
+                    description: Authentication specifies how incoming JWT's should
+                      be validated.
+                    properties:
+                      enabled:
+                        description: |-
+                          Whether to enable JWT validation.
+                          If enabled, incoming JWT's will be validated against the issuer specified in the app registration and the generated audience.
```

Nitpick: possessive vs plural
```suggestion
                         If enabled, incoming JWTs will be validated against the issuer specified in the app registration and the generated audience.
Comment
```